go.mod

```diff
@@ -7,26 +7,40 @@ require (
 	github.com/DATA-DOG/go-sqlmock v1.3.3
 	github.com/biogo/store v0.0.0-20190426020002-884f370e325d
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/coreos/etcd v3.3.13+incompatible
 	github.com/go-sql-driver/mysql v1.4.1
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/golang/groupcache v0.0.0-20191227052852-215e87163ea7 // indirect
 	github.com/google/uuid v1.1.1
+	github.com/gorilla/websocket v1.4.1 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.12.1 // indirect
+	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
-	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
-	github.com/pingcap/errors v0.11.4
+	github.com/pingcap/check v0.0.0-20191216031241-8a5a85928f12
+	github.com/pingcap/errors v0.11.5-0.20190809092503-95897b64e011
 	github.com/pingcap/failpoint v0.0.0-20200115060041-f2180fbf0df8
 	github.com/pingcap/kvproto v0.0.0-20200108025604-a4dc183d2af5
 	github.com/pingcap/log v0.0.0-20191012051959-b742a5d432e9
-	github.com/pingcap/parser v0.0.0-20191012071233-32876040fefb
-	github.com/pingcap/pd v1.1.0-beta.0.20190923032047-5c648dc365e0
-	github.com/pingcap/tidb v1.1.0-beta.0.20191017064824-e1ba309148ad
-	github.com/pingcap/tidb-tools v2.1.3-0.20190321065848-1e8b48f5c168+incompatible
-	github.com/prometheus/client_golang v0.9.0
-	github.com/remyoudompheng/bigfft v0.0.0-20190728182440-6a916e37a237 // indirect
-	github.com/sirupsen/logrus v1.4.2 // indirect
-	github.com/spf13/cobra v0.0.3
+	github.com/pingcap/parser v0.0.0-20200109073933-a9496438d77d
+	github.com/pingcap/pd v1.1.0-beta.0.20191219054547-4d65bbefbc6d
+	github.com/pingcap/tidb v1.1.0-beta.0.20200116051450-6e648401e44d
```

Comment:
hi, we have found a new bug in tidb, maybe need upgrade tidb hash after https://github.com/pingcap/tidb/pull/14631 merged thx

Reply:
ok thanks